front-end/src/utils/mutators.js

```diff
@@ -26,7 +26,7 @@ export const getFetchRequest = (
     return true;
   } catch (e) {
     console.error(`Не удалось загрузить ${errorMsg}:`, e);
-
+    this.$message.error("Не удалось загрузить данные");
```

Comment:
This is not needed, because `getError` already shows error message.
Same for changes below.